source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
++--------+-------------+----------------+----------+----------+---------------+
+| GNU    | |pe_gnu|    | ``gcc-native`` | C        | ``cc``   | ``gcc``       |
+|        |             |                +----------+----------+---------------+
+|        |             |                | C++      | ``CC``   | ``g++``       |
+|        |             |                +----------+----------+---------------+
+|        |             |                | Fortran  | ``ftn``  | ``gfortran``  |
++--------+-------------+----------------+----------+----------+---------------+
+| Intel  | |pe_intel|  | ``intel``      | C        | ``cc``   | ``icx``       |
+|        |             |                +----------+----------+---------------+
+|        |             |                | C++      | ``CC``   | ``icpx``      |
+|        |             |                +----------+----------+---------------+
+|        |             |                | Fortran  | ``ftn``  | ``ifort``     |
+|        |             +----------------+----------+----------+---------------+
+|        |             | |intel_cl|     | C        | ``cc``   | ``icc``       |
+|        |             |                +----------+----------+---------------+
+|        |             |                | C++      | ``CC``   | ``icpc``      |
+|        |             |                +----------+----------+---------------+
+|        |             |                | Fortran  | ``ftn``  | ``ifort``     |
+|        |             +----------------+----------+----------+---------------+
+|        |             | |intel_oa|     | C        | ``cc``   | ``icx``       |
+|        |             |                +----------+----------+---------------+
+|        |             |                | C++      | ``CC``   | ``icpx``      |
+|        |             |                +----------+----------+---------------+
+|        |             |                | Fortran  | ``ftn``  | ``ifx``       |
++--------+-------------+----------------+----------+----------+---------------+
+| NVIDIA | |pe_nvhpc|  | ``nvhpc``      | C        | ``cc``   | ``nvc``       |
+|        |             |                +----------+----------+---------------+
+|        |             |                | C++      | ``CC``   | ``nvc++``     |
+|        |             |                +----------+----------+---------------+
+|        |             |                | Fortran  | ``ftn``  | ``nvfortran`` |
++--------+-------------+----------------+----------+----------+---------------+
 
 .. note::
 
-  Each user has a 50 GB limit.
+    The ``gcc-native`` compiler module was introduced in the December 2023
+    release of the HPE/Cray Programming Environment (CrayPE) and replaces
+    ``gcc``. ``gcc`` provides GCC installations that were packaged within
+    CrayPE, while ``gcc-native`` provides GCC installations outside of CrayPE.
 
-Home is mounted on:
+Cray programming environment and compiler wrappers
```

Comment:
This text is repeated, see line 548 and subsequent. You decide where it fits best.

Reply:
Fixed in a new push.